spec/system/admin_index_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe 'Listing', type: :system do
+  before do
+    admin = create(:admin)
+    login_as(admin, scope: :admin)
+    driven_by(:selenium_chrome_headless)
+  end
+
+  it '#index?' do
+    visit '/admins/kudos'
+    expect(page).to have_content 'Kudos listing page'
```

Comment:
name of this spec is odd, it should be more like admin kudos index spec

also you are just checking right now if the link is there, you should create one kudo, and check if it is visible on that page